pkg/controller/kogitoapp/resource/deployment_config.go

```diff
@@ -192,6 +193,7 @@ func SetInfinispanEnvVars(cli *client.Client, kogitoInfra *v1alpha1.KogitoInfra,
 			vars := envVarInfinispanQuarkus
 			if kogitoApp.Spec.Runtime == v1alpha1.SpringbootRuntimeType {
 				vars = envVarInfinispanSpring
+				framework.SetEnvVar(envVarInfinispanSpringUseAuth, "true", &dc.Spec.Template.Spec.Containers[0])
```

Comment:
I guess we can follow the same pattern as the Quarkus map and also add the var [`QUARKUS_INFINISPAN_CLIENT_USE_AUTH`](https://quarkus.io/guides/infinispan-client#quarkus-infinispan-client_quarkus.infinispan-client.use-auth). This way this line would be outside this conditional.

Reply:
added